scripts/link-data.sh

```diff
@@ -35,6 +36,7 @@ mkdir -p machine-learning/data
 mkdir -p pathway-analysis/data
 
 link_locs=(
+  RNA-seq/QC
```

Comment:
We actually don't want to link to this whole directory, as that will result in users being unable to add their own files to it (they can't write to linked files). 

More specifically, we really don't want links to include `RNA-seq/QC/gastric-cancer/fastp/SRR585570` and `RNA-seq/QC/gastric-cancer/fastqc/SRR585570` as this would make them unable to complete the workshop preprocessing.

We could add the links to all of the other samples, as we do in the `salmon_quant` folder (I think we had that at some point before), but we can't link the root, unfortunately.

Reply:
Ah gotcha. This make sense of why it was this way then. Hmm... I don’t particularly want to link to each sample, that would make this section very long for something that might not be used. Is there an easy enough way for us to tell an interested participant how to get links these extra QC files?

Reply:
Maybe the thing to do is to point them to `~/shared-data/training-modules/RNA-seq/QC/gastric-cancer` for the multiQC command? With a note about why that strange path...

Reply:
I have to use `../../../../shared/data/training-modules/RNA-seq/QC/gastric-cancer` to get there because `~` goes to my own home directory and I assume it would be this way for the participants?

Reply:
All participant will have a link of `~/shared-data` -> `/shared/data` in their home directories

Reply:
Okay cool. :+1: